addons/binding/org.openhab.binding.dscalarm/src/main/java/org/openhab/binding/dscalarm/internal/discovery/DSCAlarmBridgeDiscovery.java

```diff
@@ -0,0 +1,166 @@
+
+	/**
+	 * Constructor.
+	 */
+	public DSCAlarmBridgeDiscovery() {
+		super(DSCAlarmBindingConstants.SUPPORTED_BRIDGE_THING_TYPES_UIDS, 15, true);
+	}
+
+	@Override
+	public Set<ThingTypeUID> getSupportedThingTypes() {
+		return DSCAlarmBindingConstants.SUPPORTED_BRIDGE_THING_TYPES_UIDS;
+	}
+
+	@Override
+	protected void startScan() {		
+		logger.trace("Start DSC Alarm Bridge discovery.");
+		scheduler.execute(envisalinkBridgeDiscoveryRunnable);	
+		scheduler.execute(it100BridgeDiscoveryRunnable);	
+	}
+
+    @Override
+    protected void startBackgroundDiscovery() {
+        logger.debug("Start DSC Alarm Bridge background discovery");
+        if (envisalinkBridgeDiscoveryJob == null || envisalinkBridgeDiscoveryJob.isCancelled()) {
+        	envisalinkBridgeDiscoveryJob = scheduler.scheduleAtFixedRate(envisalinkBridgeDiscoveryRunnable, 0, refreshInterval, TimeUnit.SECONDS);
+        }
+        if (it100BridgeDiscoveryJob == null || it100BridgeDiscoveryJob.isCancelled()) {
+        	it100BridgeDiscoveryJob = scheduler.scheduleAtFixedRate(it100BridgeDiscoveryRunnable, 0, refreshInterval, TimeUnit.SECONDS);
+        }
+    }
+
+    @Override
+    protected void stopBackgroundDiscovery() {
+        logger.debug("Stop DSC Alarm Bridge background discovery");
+        if (envisalinkBridgeDiscoveryJob != null && !envisalinkBridgeDiscoveryJob.isCancelled()) {
+        	envisalinkBridgeDiscoveryJob.cancel(true);
+        	envisalinkBridgeDiscoveryJob = null;
+        }
+        
+        if (it100BridgeDiscoveryJob != null && !it100BridgeDiscoveryJob.isCancelled()) {
+        	it100BridgeDiscoveryJob.cancel(true);
+        	it100BridgeDiscoveryJob = null;
+        }
+    }
+    
+	private Runnable envisalinkBridgeDiscoveryRunnable = new Runnable() {
+		@Override
+		public void run() {
+			envisalinkBridgeDiscovery.discoverBridge();
+		}
+	};
+
+	private Runnable it100BridgeDiscoveryRunnable = new Runnable() {
+		@Override
+		public void run() {
+			it100BridgeDiscovery.discoverBridge();
+		}
+	};
+
+	/**
+	 * Method to add an Envisalink Bridge to the Smarthome Inbox.
+	 * 
+	 * @param ipAddress
+	 */
+	public void addEnvisalinkBridge(String ipAddress) {
+		logger.debug("addBridge(): Adding new Envisalink Bridge on {} to Smarthome inbox",ipAddress);
+		
+		String bridgeID = ipAddress.replace('.', '_');
+		Map<String, Object> properties = new HashMap<>(0);
+		properties.put(EnvisalinkBridgeConfiguration.IP_ADDRESS, ipAddress);
+		
+		try {
+			ThingUID thingUID = new ThingUID(DSCAlarmBindingConstants.ENVISALINKBRIDGE_THING_TYPE, bridgeID);
+			
+			if (thingUID != null) {
+	
+				DiscoveryResult result = DiscoveryResultBuilder.create(thingUID).withProperties(properties) //.withProperty(EnvisalinkBridgeConfiguration.IP_ADDRESS, ipAddress)
+						.withLabel("EyezOn Envisalink Bridge - " + ipAddress).build();
+				thingDiscovered(result);
+				
+				logger.debug("addBridge(): '{}' was added to Smarthome inbox.",result.getThingUID());
```

Comment:
those things are logged by the framework already, so either remove it or reduce to TRACE.